pkg/operator/ceph/cluster/osd/pod.go

```diff
@@ -58,7 +58,7 @@ func (c *Cluster) makeJob(nodeName string, devices []rookalpha.Device,
 
 	job := &batch.Job{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf(prepareAppNameFmt, nodeName),
+			Name:      k8sutil.TruncateName(fmt.Sprintf(prepareAppNameFmt, nodeName)),
```

Comment:
Do we need to look at truncating all the names for resources dependent on node name? There are a few configmaps based on the node name as well.

Reply:
Yes, I'll look into finding other places where this needs to happen.